### ISSUE TITLE
Settings > Text > Position settings doesn't save, but do apply

### ISSUE THREAD

**Issue by aidenmitchell** [OrangeJedi/Aerial#23]
**Describe the bug**
When changing text position settings, after setting them and exiting settings, then opening settings, the text positions settings reset to default, but they do apply once the screen saver starts.

**To Reproduce**
1. Go to Settings
2. Click on Text
3. Chick on Position
4. Apply settings
5. Exit settings
6. Open settings and view Text > Position

**Expected behavior**
Previous set settings are shown.

**Aerial Version**
Beta 0.5.2

**Comment by OrangeJedi** [comment]
I'm having trouble reproducing this bug. What settings are you applying when you set up the text? And is no information being shown, or are just parts of it missing? Any other information or screenshots would be nice. I hope we can get this fixed soon.

**Comment by aidenmitchell** [comment]
After playing around with it, I think I just misunderstood how the UI was designed. I've attached a video of what I meant, but I discovered that clicking on the radio button to select the position will show previously applied settings. I just expected it to show that without having to pick a position, but the way it is currently I can have more than one thing in more than one position, which is better.

Video: https://streamable.com/rsj6ai
